test/parallel/test-crypto-dh.js

```diff
@@ -277,3 +281,8 @@ ecdh5.setPrivateKey(cafebabeKey, 'hex');
   // Verify object state did not change.
   assert.strictEqual(ecdh5.getPrivateKey('hex'), cafebabeKey);
 });
+
+// invalid test: curve argument is undefined
+assert.throws(() => {
+  crypto.createECDH();
+}, /"curve" argument should be a string/);
```

Comment:
Can you change this line to `/^TypeError: "curve" argument should be a string$/`